### PR TITLE
fix(sourcing): Rm sourcing

### PR DESCRIPTION
**Pull request by ingkebil**
This PR prepares the deliver script to run bash functions instead of bash aliases.

How to test?
1. install on stage: `bash update-deliver-stage.sh checkfornewdemux-hasta`
1. install servers on stage: `bash update-servers-stage.sh`
1. `rm /home/proj/stage/demultiplexed-runs/181218_ST-E00266_0336_BHVTWHCCXY/delivery.txt`
1. run: `source /home/proj/stage/servers/resources/hasta.scilifelab.se/usestage.sh`
1. run: `bash /home/proj/stage/bin/git/deliver/scripts/checkfornewdemuxonhasta.bash`

Expected outcome:
1. exit code 0 :)
1. in ` /home/proj/stage/demultiplexed-runs/181218_ST-E00266_0336_BHVTWHCCXY` the stats-* files should be newly created and not zero sized.

Sign offs:
- [x] code review by @barrystokman 
- [x] testing done by @emiliaol 
- [x] approval for deploy by @ingkebil

This is minor version bump: we extend the functionality. We have also changed how one parameter is passed into the script as well as simplified how the activate environment was detected. Both changes would only warrant a patch change.

### CODE REVIEW

**Comment by ingkebil** [comment]
Sorry, I forgot to add the sign off in the PR description! Did you test the code or did you perform a code review? :)

**Comment by barrystokman** [comment]
Emilia is testing it right now! On her day off no less!

**Comment by emiliaol** [comment]
Tested it according to the test case. The stats-files creation works perfectly but the emailing does not (suggested some changes but I haven't tested them):
Skärmavbild 2019-06-19 kl  09 14 30

File: scripts/checkfornewdemuxonhasta.bash [hunk 1]
```diff
@@ -1,15 +1,12 @@
...
-source ${HOME}/.bashrc
-useprod
-
 ########
 # VARS #
 ########
 
 HASTA_DEMUXES_DIR=${1-${PROJECT_HOME}/${ENVIRONMENT}/demultiplexed-runs/}
 ERROR_EMAIL=${2-clinical-demux@scilifelab.se}
+MAILTO=${clinical-demux@scilifelab.se}
```
Comment (emiliaol) [inline scripts/checkfornewdemuxonhasta.bash]:
```suggestion
MAILTO=${2-clinical-demux@scilifelab.se}
```

**Comment by emiliaol** [comment]
Also tested the email changes and it seems to work now except that the email isn't delivered to the gmail, but that seems to be a bigger problem also happening for microSALT:
Skärmavbild 2019-06-19 kl  09 33 10

**Comment by ingkebil** [comment]
Thanks for testing!

**Pull request merged**